tests/test_controller.py

```diff
@@ -2,28 +2,27 @@
 import pytest
 from mock import MagicMock
 from seedsigner.controller import Controller
-from seedsigner.models.settings import Settings
 from seedsigner.models.settings_definition import SettingsConstants
 
 
 
-def test_singleton_init_fails():
+@pytest.fixture(scope="module")
+def reset_controller():
+    Controller._instance = None
+    Controller.configure_instance(disable_hardware=True)
+    yield Controller
```

Comment:
I'm not as familiar with the `yield` keyword but is it necessary in this case? Once the class has been reset and configured, I don't think we need to yield/return anything from the fixture.